python/lib/adaguc/runAdaguc.py

```diff
@@ -101,16 +101,13 @@ def getAdagucEnv(self, adagucenv = {}):
 
         return adagucenv
 
-    def updateLayerMetadata(self):
+    async def updateLayerMetadata(self):
```

Comment:
This resolves the issue.

Can you bump the version in Definitions.h, in the Dockerfile and write a short section in News.md?